pkg/kubelet/rkt/rkt.go

```diff
@@ -82,7 +84,7 @@ const (
 // runtime implements the Containerruntime for rkt. The implementation
 // uses systemd, so in order to run this runtime, systemd must be installed
 // on the machine.
-type runtime struct {
+type Runtime struct {
```

Comment:
Maybe I'm missing something here...but why do we have to capitalize "runtime" for this PR?